src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,53 @@
+package calculator;
+
+import org.junit.jupiter.api.Test;
+
+import static calculator.Calculator.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class CalculatorTest
+{
+    // 1. 빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.(예 : “” => 0, null => 0)
+    @Test
+    void null_빈문자열_값()
+    {
+        assertThat(cal(null)).isEqualTo(0);
+        assertThat(cal("")).isEqualTo(0);
+    }
```

Comment:
```suggestion
    @DisplayName("빈 문자열 또는 null 값을 입력할 경우 0을 반환해야 한다.")
    @ParameterizedTest(name = "입력값: {0}")
    @NullAndEmptySource
    void null_빈문자열_값(String input)
    {
        assertThat(cal(input)).isEqualTo(0);
    }
```
이런 방법도 있으니 한번 돌려보세요!